### ISSUE TITLE
Rozvržení prvků u defibrilátoru a tlaku

### ISSUE THREAD

**Issue by kanakl** [PETER-project/monitor_vital_functions#97]
Tak jsem po delší době opět zapnul monitor a jen tak jsem si to proklikával a u panelů defibrilátoru a tlaku jsem byl možná pro intuitivnější ovládání předělal zavírání panelů. Přidal bych křížek do pravého rohu, jak je u defibrilátoru tlačitko pro ztlumení a odebral bych ho ze spodní části. To samé bych provedl u tlaku, kde bych přesunul tlačítko pro zlumení zvuku(možná řešit jenom ikonou) i pro zavření okna. Souhlasíte? Je to sice detail, ale celkem mě zmátlo uzavírání toho okna :)

**Comment by pornosaur** [comment]
Jo, jsem pro. Jenom, co jsem na to koukal, tak by nebylo na skodu uzavirat ty panely pri prekliku na jiny. 

-Priklad:
   - Otevru defibrilator, nasledne tlak a potom uzavru tlak, tak pod nim je furt defibrilator, takze ocekevama, ze je tam aktivni jenom prekryty tim tlakem, coz by urciten nemelo byt.

**Comment by kanakl** [comment]
Jo taky dobrá poznámka.

**Comment by pornosaur** [comment]
Mi jeste napadlo, jestli by to spis nechtelo probrat s Kunaskem, prece jenom to jakoby neni na nas...
Akorat ma ted asi dovcu, protoze mi uz dlouho neodepsal.

**Comment by kanakl** [comment]
jo to bude asi nejlepší
